### PR TITLE
constants: Update URLs

### PR DESCRIPTION
**Pull request by regexowl**
This updates documentation URLs.

### CODE REVIEW

**Comment by regexowl** [comment]
This is a follow up to https://github.com/osbuild/image-builder-frontend/pull/2208 based on a suggestion made by @swadeley https://github.com/osbuild/image-builder-frontend/pull/2208#issuecomment-2208650371

File: src/constants.ts [hunk 1]
```diff
@@ -28,18 +28,18 @@ export const ACTIVATION_KEYS_PROD_URL =
...
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/security_guide/chap-compliance_and_vulnerability_scanning';
 export const CREATING_IMAGES_WITH_IB_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index';
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index';
 export const MANAGING_WITH_DNF_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/managing_software_with_the_dnf_tool/index';
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/managing_software_with_the_dnf_tool/index';
 export const CREATING_IMAGES_WITH_IB_SERVICE_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_rhel_images_using_the_image_builder_service';
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/creating_customized_rhel_images_using_the_image_builder_service';
```
Comment (regexowl) [inline src/constants.ts:37]: This link seems to be dead, @ochosi can I replace it with `https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index` (CREATING_IMAGES_WITH_IB_URL) or is there a better link we could use?

image

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

File: src/constants.ts (hunk 1 above)
Comment (regexowl) [reply src/constants.ts:37]: All is good, I haven't refreshed and missed this PR https://github.com/osbuild/image-builder-frontend/pull/2211 from @croissanne :sweat_smile:

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Review by ezr-ondrej — APPROVED** [review]
:tada:

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Pull request merged**